installer/bundle/bundle_skel_Linux.sh

```diff
@@ -617,14 +602,21 @@ case "$installMode" in
         echo "Updating cross-platform agent ..."
```

Comment:
It looks like "I" and "U" are very very close (but not 100% identical). Perhaps better to merge the two into one case for code reduction?